common/app/uiComponent/core/RenderingActor.scala

```diff
@@ -0,0 +1,23 @@
+package uiComponent.core
+
+import akka.actor.Actor
+import model.ApplicationContext
+import uiComponent.UIComponent
+import scala.util.Try
+
+case class Rendering(component: UIComponent, ac: ApplicationContext)
+
+case class RenderingException(error: String) extends RuntimeException(error)
+
+class RenderingActor extends Actor with JavascriptRendering {
+
+  override def javascriptFile: String = "ui.bundle.server.js"
```

Comment:
Potentially looking at the tests might help more. Right now the serverside part doesn't require `ui.bundle.server.js` to be any more complicated then https://github.com/guardian/frontend/pull/17515/files#diff-fa7954d48ee0602561496f1794e06217R1